references/tags.tsv

```diff
@@ -20,6 +22,7 @@ Bracken2016_mirna	doi:10.1038/nrg.2016.134
 Boza	arxiv:1603.09195
 Buggenthin2017_imaged_lineage	doi:10.1038/nmeth.4182
 Caruana2014_need	arxiv:1312.6184
+Caruana2015_intelligible	https://dl.acm.org/citation.cfm?id=2788613
```

Comment:
`https://dl.acm.org/citation.cfm?id=2788613` to `url:https://dl.acm.org/citation.cfm?id=2788613`